File: 151004/Matsiushyn/Lab2/DC_Lab1/DTO/LabelResponseTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class LabelResponseTo(int Id, string? Name) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public string? name { get; set; } = Name;
    }
}

File: 151002/Luschickij/fullProject/DC_lab/publisher/src/main/java/com/luschickij/publisher/dto/label/LabelResponseTo.java
package com.luschickij.publisher.dto.label;

import com.fasterxml.jackson.annotation.JsonRootName;
import com.luschickij.publisher.model.IdentifiedEntity;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.math.BigInteger;

@EqualsAndHashCode(callSuper = true)
@Data
@Builder
@JsonRootName("label")
@AllArgsConstructor
@NoArgsConstructor
public class LabelResponseTo extends IdentifiedEntity {

    private Long id;
    private String name;

}
